app/main.py

```diff
@@ -1,3 +1,86 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from app.shop import Shop
+from app.customer import Customer
```

Comment:
Please be more accurate with imports. It should be blank line between built-in and your own imports and in each block they should be sorted in alphabetical order